vpd_tool_impl.cpp

```diff
@@ -949,91 +948,82 @@ void VpdTool::parseSVPDOptions(const nlohmann::json& json)
 
                 for (const auto& data : recKwData)
                 {
-                    do
```

Comment:
do not remove do while loop as it hits the default case. So change the switch case at line 997 to if else.